envi/__init__.py

```diff
@@ -1529,31 +1535,15 @@ def loadModuleFromPath(modname, modpathtup):
     '''
     Load a Python module given a module path
     '''
-    modpath = os.sep.join(modpathtup)
-
-    # if we hand in the path to the directory, load the __init__.py
-    if os.path.isdir(modpath):
-        modpath = os.sep.join([modpath, '__init__.py'])
-
-    if not os.path.exists(modpath):
-        raise ModuleLoadFailure(modname, "Path does not exist: %r" % modpath)
-
-    # get the module spec
-    spec = imputil.spec_from_file_location(modname, modpath)
-    if not spec:
-        raise ModuleLoadFailure(modname, "Failed to load module")
-
-    # create an unintialized module from the spec
-    module = imputil.module_from_spec(spec)
-    if not module:
-        raise ModuleLoadFailure(modname, "Failed to create uninitialized module from the spec")
+    modname = '.'.join(modpathtup)
+    try:
+        module = importlib.import_module(modname)
+    except ModuleNotFoundError as exc:
+        raise ModuleLoadFailure(modname, e.msg)
```

Comment:
fixed upstream.  it doesn't look like this, but the problem is resolved.

Reply:
I will take the upstream version when I rebase.

Reply:
i've merged in v1.1.0 into envi_ppc.